rst_files/getting_started.rst

```diff
@@ -270,38 +244,39 @@ Documentation should now appear in the browser
 Other Content
 ^^^^^^^^^^^^^^^
 
-In addition to executing code, the Jupyter notebook allows you to embed text, equations, figures and even videos in the page
+In addition to executing code, the Jupyter notebook allows you to embed text, 
+equations, figures and even videos in the document
 
 For example, here we enter a mixture of plain text and LaTeX instead of code
 
 .. figure:: /_static/figures/nb6_julia.png
-   :scale: 100%
+   :scale: 80%
```

Comment:
These screenshots should be zoomed out and include the 'In:' 'Out:' flags because they were kind of confusing to interpret

Reply:
I think this is a good time to go through all of the screenshots to see what requires a zoom-in/etc.